packages/kbn-optimizer/limits.yml

```diff
@@ -143,7 +144,7 @@ pageLoadAssetSize:
   snapshotRestore: 79032
   spaces: 57868
   stackAlerts: 58316
-  stackConnectors: 52131
+  stackConnectors: 67227
```

Comment:
Could you help me understand why this limit needs to be increased?

Reply:
I ran the built-in script to update the limits and this got updated too, I guess that part of the codebase didn't it the limit yet but went over a threshold maybe so it got updated, but I'm not really sure if that's the reason, only the script knows 😅

Reply:
Ah, makes sense. Do we want to keep this or revert this specific line?

Reply:
If the scripts believe this should be updated I would trust it :)